aws-serverless-java-container-core/src/main/java/com/amazonaws/serverless/proxy/internal/servlet/AwsProxyHttpServletResponseWriter.java

```diff
@@ -73,7 +73,7 @@ public AwsProxyResponse writeResponse(AwsHttpServletResponse containerResponse,
 
         awsProxyResponse.setStatusCode(containerResponse.getStatus());
 
-        if (containerResponse.getAwsProxyRequest() != null && containerResponse.getAwsProxyRequest().getRequestSource() == AwsProxyRequest.RequestSource.ALB) {
+        if (containerResponse.getAwsProxyRequest() != null && containerResponse.getAwsProxyRequest().getRequestSource() == AwsProxyRequest.RequestSource.ALB && Response.Status.fromStatusCode(containerResponse.getStatus()) != null) {
```

Comment:
Thanks for the pull request.

Can we please move `Response.Status.fromStatusCode(containerResponse.getStatus())` to a variable instead of doing the lookup twice?

Apart from that it looks good. Will merge afterwards and include in the 1.10 release